joint_names:
  - "spine_base"
  - "spine_mid"
  - "neck"
  - "head"
  - "shoulder_left"
  - "elbow_left"
  - "wrist_left"
  - "hand_left"
  - "shoulder_right"
  - "elbow_right"
  - "wrist_right"
  - "hand_right"
  - "hip_left"
  - "knee_left"
  - "ankle_left"
  - "foot_left"
  - "hip_right"
  - "knee_right"
  - "ankle_right"
  - "foot_right"
  - "spine_shoulder"
  #- "hand_tip_left"
  #- "thumb_left"
  #- "hand_tip_right"
  #- "thumb_right"

edges:
  - "spine_base"
  - "spine_mid"
  - "spine_mid"
  - "spine_shoulder"
  - "spine_shoulder"
  - "neck"
  - "neck"
  - "head"
  - "spine_shoulder"
  - "shoulder_left"
  - "shoulder_left"
  - "elbow_left"
  - "elbow_left"
  - "wrist_left"
  - "wrist_left"
  - "hand_left"
  - "spine_shoulder"
  - "shoulder_right"
  - "shoulder_right"
  - "elbow_right"
  - "elbow_right"
  - "wrist_right"
  - "wrist_right"
  - "hand_right"
  - "spine_base"
  - "hip_left"
  - "hip_left"
  - "knee_left"
  - "knee_left"
  - "ankle_left"
  - "ankle_left"
  - "foot_left"
  - "spine_base"
  - "hip_right"
  - "hip_right"
  - "knee_right"
  - "knee_right"
  - "ankle_right"
  - "ankle_right"
  - "foot_right"
  #"left_hand", "hand_tip_left",
  #"left_hand", "thumb_left",
  #"right_hand", "hand_tip_right",
  #"right_hand", "thumb_right"

radius:
  spine_base: 0.05
  spine_mid: 0.05
  neck: 0.05
  head: 0.05
  shoulder_left: 0.05
  elbow_left: 0.05
  wrist_left: 0.05
  hand_left: 0.05
  shoulder_right: 0.05
  elbow_right: 0.05
  wrist_right: 0.05
  hand_right: 0.05
  hip_left: 0.05
  knee_left: 0.05
  ankle_left: 0.05
  foot_left: 0.05
  hip_right: 0.05
  knee_right: 0.05
  ankle_right: 0.05
  foot_right: 0.05
  spine_shoulder: 0.05
